x/pairing/keeper/pairing.go

```diff
@@ -105,12 +106,11 @@ func (k Keeper) GetPairingForClient(ctx sdk.Context, chainID string, clientAddre
 
 // function used to get a new pairing from provider and client
 // first argument has all metadata, second argument is only the addresses
-func (k Keeper) getPairingForClient(ctx sdk.Context, chainID string, clientAddress sdk.AccAddress, block uint64) (providers []epochstoragetypes.StakeEntry, allowedCU uint64, legacyStake bool, errorRet error) {
-	var geolocation uint64
-	var providersToPair uint64
+func (k Keeper) getPairingForClient(ctx sdk.Context, chainID string, clientAddress sdk.AccAddress, block uint64) ([]epochstoragetypes.StakeEntry, uint64, bool, error) {
```

Comment:
why did we remove the names? now it is not understood what the function returns

Reply:
you can understand the return values meaning by looking at calls for it.
99% of our code doesn't have named return values, so I thought this should be fixed.
I can bring back the names if you think they're necessary

Reply:
if a function returns more than an arg and an error it is hard to understand from the function name what it returns unless you go inside and reverse engineer the code

Reply:
done